jkube-kit/jkube-kit-spring-boot/src/main/java/org/eclipse/jkube/springboot/SpringBootLayeredJar.java

```diff
@@ -65,10 +68,16 @@ public String getMainClass() {
   }
 
   public List<String> listLayers() {
-    final LayerListCommand layerListCommand = new LayerListCommand(kitLogger, layeredJar);
-    try {
-      layerListCommand.execute();
-      return layerListCommand.getLayers();
+    try (JarFile jarFile = new JarFile(layeredJar)) {
+      String layerFileContent = IOUtils.toString(jarFile.getInputStream(jarFile.getEntry("BOOT-INF/layers.idx")), StandardCharsets.UTF_8);
+      List<Map<String, List<String>>> layers = Serialization.unmarshal(layerFileContent, List.class);
```

Comment:
Why aren't we using Serialization.unmarshal from InputSream?

Reply:
oh, okay. let me use that method.